### ISSUE TITLE
Guides may require a proofread?

### ISSUE THREAD

**Issue by crispen-smith** [compoundjs/guides#20]
Hi!
I'm sorry if this is out of line but I seem to have noticed a pattern with the basics/crash-course.md file where phrases are missing the word "the", for example "Let's start with routes, because this is first place in the compound stack",  my feeling is that this might read better as "Let's start with routes, because this is _the_ first place in the compound stack".

From what I've read so far the compoundjs may soon become my new goto back-end framework, I'd be happy to do a pull request with some proofreading done on the crash-course.md file.

**Comment by anatoliychakkaev** [comment]
Sure, go ahead. Thanks!

**Comment by crispen-smith** [comment]
I'm working on it now, it's a fairly thorough proof read, so I'll send you a note when I've complete to see if you have any thoughts before I submit a pull.

In working on it I realized how much I appreciate the details that you've put into this.  I wasted a day on another framework because this simple line wasn't included in the docs:

> In development mode every modification of an existing model, controller or view 
> file will be updated automatically, but when you modify routes or schema you
> have to restart the server manually. Alternatively, use the `node-dev` command (`npm install
> node-dev` first) to restart server automatically on every file change.

(I copied after I proposed the edit to this block).  

I came to node having written front-end javascript for several years so my conception was that the whole thing would be reloaded on every refresh, putting a warning about this right in the docs is a brilliant move.

**Comment by crispen-smith** [comment]
I've made a first pass over at: https://github.com/crispen-smith/guides/blob/master/basics/crash-course.md

also, the intro blurb for the crash course at: https://github.com/crispen-smith/guides/blob/master/README.md

**Comment by 1602** [comment]
Thanks man, I reviewed your work and I think you doing it great. Do not hesistate add your own thoughts, because it's very important to see framework from the different points. My own vision: everything is obvious, but of course for newcomer it isn't true, that's why it's good if newcomer will contrubute and clarify some unclear parts.

Good luck!

**Comment by crispen-smith** [comment]
That's my thinking as well.  It's difficult for a code author to write docs
that are intended for anything other than "This is how this function works,
and why it works this way" sort of docs, and sometimes even those are hard
because it _just makes sense_.

Writing docs that are intended for someone with no understanding of a
module is very difficult and I think you've done a fantastic job, I'm just
hoping that my polishing will push it over the edge.

I do have a couple of questions about specific sections, but those are
about me not 100% understanding.  I need to switch tacks for the day but I
will come back to this tomorrow, do a second pass and put my questions
together for you.

**Comment by crispen-smith** [comment]
I'm super frustrated with myself for not getting back onto this sooner... August turned into a really deep development cycle where I hardly came up for air.

I'm hoping to start touching this again in the next day or two.

I'd sent you an email to the address you gave me, about mid-month, did you see it or did I get spamfiltered?

**Comment by 1602** [comment]
Sorry man, I haven't noticed it in my inbox, because it was busy days, as
I'm moving to London now and have a lot of things to sort out.

On Fri, Aug 30, 2013 at 3:13 AM, Crispen Smith notifications@github.comwrote:

> I'm super frustrated with myself for not getting back onto this sooner...
> August turned into a really deep development cycle where I hardly came up
> for air.
> 
> I'm hoping to start touching this again in the next day or two.
> 
> I'd sent you an email to the address you gave me, about mid-month, did you
> see it or did I get spamfiltered?
> 
> —
> 
> Reply to this email directly or view it on GitHubhttps://github.com/compoundjs/guides/issues/20#issuecomment-23536743
> .

## 

Thanks,
Anatoliy Chakkaev

**Comment by crispen-smith** [comment]
No worries.  I'd been more worried about it being eaten by your spam filters than anything else.

I hope the London move is driven by good things.

**Comment by crispen-smith** [comment]
Heya,

Sorry I've been so quiet on my end the last couple of months.  Went through some fairly crippling life events.  However, all seems to be sorted now.

Is there still value in my reviewing docs?

**Comment by 1602** [comment]
Yep, this is always actual task. Also if you have time and want to try our
challenge -- feel free to join our team on NKO2013: http://1602.ws/nko.html

On Wed, Nov 6, 2013 at 2:24 AM, Crispen Smith notifications@github.comwrote:

> Heya,
> 
> Sorry I've been so quiet on my end the last couple of months. Went through
> some fairly crippling life events. However, all seems to be sorted now.
> 
> Is there still value in my reviewing docs?
> 
> —
> 
> Reply to this email directly or view it on GitHubhttps://github.com/compoundjs/guides/issues/20#issuecomment-27835676
> .

## 

Thanks,
Anatoliy Chakkaev

**Comment by crispen-smith** [comment]
I'm definitely up for working on the docs for CompoundJS...  I think it's a
great project and I know that I can offer a lot of support there.

As far as committing any sort of time to another project, I'm interested
but I need to know what your expectations are... I'm currently booked about
80 hours a week.

**Comment by anatoliychakkaev** [comment]
Nko is 48hour challenge. Details by link.
